app/workers/hg/message_worker.rb

```diff
@@ -28,10 +28,8 @@ def perform(user_id, redis_namespace, bot_class_name)
         message = Facebook::Messenger::Incoming::Message.new(raw_message)
         
         # Instantiate chatbase client
-        if ENV['CHATBASE_API_KEY']
-          @client = ChatbaseAPIClient.new
-        end
-        
+        @client = ChatbaseAPIClient.new if ENV['CHATBASE_API_KEY']
```

Comment:
Not a big deal right now, but typically you'll want to include what this is a client for in the name of the variable itself: `@chatbase_client`